app/graphql/mutations/update_aid_request.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Mutations
+  class UpdateAidRequest < BaseMutation
+    type Types::AidRequestType
+    argument :id, ID
+    argument :aid_type, String, required: false
+    argument :language, String, required: false
+    argument :description, String, required: false
+    argument :status, String, required: false 
+
+    def resolve(id:, **args)
+      begin
+      aid_request = AidRequest.find(id)
+      rescue ActiveRecord::RecordNotFound => e
+        return GraphQL::ExecutionError.new("Invalid input: #{e.message}")
+      end
+
+      if args.values.any?(&:blank?)
```

Comment:
Language is the only optional one (since it's only relevant on language-type requests) so if an organization is updating a request to remove the "language" designation then that would be a situation where it can be blank without throwing an error. 
So maybe a conditional to make sure that blank language is accepted, but other blank fields will throw an error?

Reply:
OR: we discuss with FE to add a conditional to set `language: "N/A"` if the organization-user deletes the field while updating the request, so that they still send a string to BE. Which means that upon creation, we'd also need FE to set `language: "N/A"` if there's no input, before sending it to BE. Future conversation, though, considering the time frame.